duckduckgo/iOS
# Use design system fonts throughout NetP

Task/Issue URL: https://app.asana.com/0/0/1206056835046943/f

**Description**:

I had some design feedback about the footer fonts being inconsistent. I realised that I’d not been using the `DesignResourceKit` defined design system fonts. So I’ve updated all the NetP views to use them.

**Steps to test this PR**:
1. Just check nothing looks weird on the NetP screens. Consider checking against the alpha. The only visible change should be the footer fonts.

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
